builder/image/bin/boot

```diff
@@ -41,13 +41,13 @@ function etcd_safe_mkdir {
 etcd_safe_mkdir $ETCD_PATH/users
 
 # wait for confd to run once and install initial templates
-until confd -onetime -node $ETCD --confdir /app --interval 5 --quiet; do
+until confd -onetime -node $ETCD --confdir /app --log-level warn --watch; do
```

Comment:
Do `-onetime` and `--watch` conflict? Seems like `-onetime` would override here.